example/vectorAdd/src/main.cpp

```diff
@@ -124,13 +132,15 @@ auto main()
     BufHost bufHostC(alpaka::mem::buf::alloc<Data, Idx>(devHost, extent));
 
     // Initialize the host input vectors A and B
-    Data * const pBufHostA = alpaka::mem::view::getPtrNative(bufHostA);
-    Data * const pBufHostB = alpaka::mem::view::getPtrNative(bufHostB);
+    Data * const pBufHostA(alpaka::mem::view::getPtrNative(bufHostA));
+    Data * const pBufHostB(alpaka::mem::view::getPtrNative(bufHostB));
     Data * const pBufHostC(alpaka::mem::view::getPtrNative(bufHostC));
+
     for (Idx i(0); i < numElements; ++i)
     {
-        pBufHostA[i] = static_cast<Data>(std::rand());
-        pBufHostB[i] = static_cast<Data>(std::rand());
+      pBufHostA[i] = static_cast<Data>(std::rand());
+      pBufHostB[i] = static_cast<Data>(std::rand());
+      pBufHostC[i] = 0;
```

Comment:
Do we have to initialize and copy C? It should be overwritten by the kernel anyway.